### DISCUSSION TITLE
Changed username

### DISCUSSION THREAD

**Discussion by porterbhall** [guysoft/FullPageOS#584] [Q&A] [answered]
I changed the user name on my Raspberry Pi. When I start FullPageOS, I get this error onscreen: ``Xsession: unable to launch "/home/pi/scripts/start_gui"``

This is because there is no /home/pi directory on the machine. I see the start_gui script in my account. Where can I change the call for that script?

**Comment by guysoft** [comment]
Hey, I am working to solve this and have a solution here:
https://github.com/guysoft/CustomPiOS/commit/c85213f2de73bd106607382fe5bac7996e658398

and
https://github.com/guysoft/FullPageOS/commit/73a536aa9267d3bdc9a75b934dc543b01fa31e99

Ill tidy it up and release another release candidate later on.

**Reply by porterbhall** [reply]
Thanks! How do I pick up the change? Do I have to reimage my card or will I pick it up as a package update?

**Reply by guysoft** [reply, marked as answer]
You need to reflash, thees is no upgrade mechanism.

**Reply by porterbhall** [reply]
Thanks. I refreshed using the nightly build and that worked.
